client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js

```diff
@@ -371,7 +372,7 @@ class DetailedProgramSplit extends React.Component {
                   bottom: 30,
                   left: 215,
                 }}
-                graph_height={divHeight}
+                graph_height={divHeight > 100 ? divHeight : "90px"}
```

Comment:
Took a look, `divHeight` is used for two things, right? `graph_height` and also `legendOffsetY`. In the case that `divHeight <= 100`, isn't this desyncing `graph_height` and `legendOffsetY`? I assume they should stay in sync, seeing as they're both derived from `divHeight`.

Especially if I'm right about that, but more or less in general too, this feels more like it should be a minimum value baked in to the declaration of `divHeight`. Declaring `divHeight` a bunch of lines up and then enforcing a minimum non-locally, down where it's actually used, could obscure from future developers that the value has a lower bound.

Reply:
Unfortunately keeping it in sync throws the legend off even more haha 
![image](https://user-images.githubusercontent.com/25855114/114921502-9c9e4380-9df8-11eb-8626-cfc570d4620a.png)

Gonna keep playing around with it...

Reply:
Strange, must be some magic in those calculations, ugh.

Reply:
Yea this is gonna be a really annoying fix, we'd have to make the chart treat all bars consistent in height (is this even possible?) and we'd also have to go in and manually render each bar after that. But theoretically, we if could do the former, the latter would be solved automatically. I tried to use the layers prop but then I realized that the bars would probably start overlapping since i'm guessing the chart itself isn't gonna scale to fit the bars

I think this method would be a good temporary fix on the small charts. I've left a comment stating that it's hacky and left a todo to refactor it.